src/content/reference/gsctl/_index.md

```diff
@@ -3,6 +3,7 @@ title = "gsctl Reference"
 description = "Documentation on gsctl, the Giant Swarm command line utility to create and delete clusters, create key pairs and more."
 date = "2018-09-11"
 type = "page"
+layout = "gsctl-reference"
```

Comment:
This applies a specific template (the new `src/layouts/section/gsctl-reference.html`) to render this page